portfolio/main/models.py

```diff
@@ -46,6 +46,15 @@ def entries(self):
 
         return entries
 
+    def featured_entries(self):
```

Comment:
Per your questions in the description, I do not see this as repetitive. Breaking out functionality in this way is a common pattern. A few tests for both functions `entries` and `featured_entries` should be added. I'm fine if you want to do that in a separate PR.